lib/appsignal/phoenix/channel.ex

```diff
@@ -51,7 +51,7 @@ if Appsignal.phoenix? do
     def channel_action(module, name, %Phoenix.Socket{} = socket, function) do
       alias Appsignal.Transaction
 
-      transaction = Transaction.start(Transaction.generate_id(), :background_job)
+      transaction = Transaction.start(Transaction.generate_id(), :channels)
```

Comment:
The namespace names are usually singular form, so we should probably keep that similar.
Since `:channels` (and `:action_cable` https://github.com/appsignal/appsignal-ruby/pull/275) will be an internal namespace name (like `:http_request` and `:background_job`) we can change the display name for that in the UI

So that would become a mapping like so:

- `http_request` => "Web"
- `background_job` => "Background"
- `action_cable` => "Action Cable" / "Channels"
- `channel` => "Channels"